### ISSUE TITLE
[Feature Request] Settings: "Cover Images" Directory, to automatically set cover images

### ISSUE THREAD

**Issue by HeyItsLollie** [Rosalie241/RMG#83]
It'd be great if game entries could automatically load cover images from a specific folder, provided the image shares the same filename as the ROM file.

eg: If "GoldenEye 007 (USA).z64" had no cover art, RMG could search in the "Cover Images\\" directory and try to match with "GoldenEye 007 (USA).png/jpg"

**Comment by Rosalie241** [comment]
Why would you need a custom directory for that? i.e on a portable installation it uses the `Covers` directory and on i.e flatpak/a system linux installation it uses the `user data path/Covers` directory.

It checks for the following files when loading the covers:
1) `MD5.png/jpg/jpeg`
2) `Good Name.png/jpg/jpeg`
3) `Internal ROM Name.png/jpg/jpeg`

(You can find all this information about a certain ROM when you right click -> ROM Information)

If you join the discord server you'll find a cover pack for lots of games available for download so you can see how it's used there.

**Comment by HeyItsLollie** [comment]
Ahaaa, so two things happened:
1. I originally searched around online (via google, not on discord) for any info regarding cover images in RMG, but couldn't find any info anywhere. RMG doesn't have any documentation readily available either, so I didn't know what to expect here.
2. I looked in RMG's installation folder _before_ attempting to manually add a cover image, and I couldn't find a `Covers` folder anywhere - not even inside the `Data` folder. It doesn't exist in the portable installation zip either.

Now that I've added a cover image myself, I see that the `Covers` folder _has_ been created.

I'd like to suggest instead that an empty `Data\Covers` directory should exist for new installations, since this isn't really obvious for new users.

**Comment by Rosalie241** [comment]
I agree documentation is lacking and should exist to explain things like this clearly, I'll keep it in mind and try to write something in the near future!

**Comment by Retroboy85** [comment]
I am new to Steam Deck and Linux OS, I downloaded the FlatPak image to my SD Card in the hope that I can simply run the emulator off my SD card rather than have it run from the internal memory.

I have it running solely from my SD card, but it took me forever to locate the correct directory to then copy over the cover art work.

I find this OS a nightmare as it places files/directories all over the place - making it very hard to find the actual location you need to be in.

So, after looking and looking, I found that placing the Covers in this directory worked for me:
/home/deck/.local/share/RMG

Now cover art automatically loads when I run RMG emulator.

Now all I need to figure out is how to get the EMULATOR (not game) to open in full screen when in Game Mode. I have tried setting this in the emulator properties
-bigpicture -fullscreen

Which worked on other emulators, but I cant find how to do it for RMG.

I have tried searching but all I find are results for either the game itself or using the likes of RetroArch or EmuDeck - which I am not using, I am using the actual emulator flatpak itself.

**Comment by Rosalie241** [comment]
`--fullscreen` will launch RMG in fullscreen
